src/components/screens/dashboard/root.css

```diff
@@ -0,0 +1,46 @@
+#root {
```

Comment:
Is this just vite boiler plate styling- if so it can be removed

Reply:
Removed